README.md

```diff
@@ -1,152 +1,93 @@
 # Label-free prediction of three-dimensional fluorescence images from transmitted-light microscopy
+
 [![Build Status](https://travis-ci.org/AllenCellModeling/pytorch_fnet.svg?branch=master)](https://travis-ci.org/AllenCellModeling/pytorch_fnet)
 [![Documentation Status](https://readthedocs.org/projects/pytorch-fnet/badge/?version=latest)](https://pytorch-fnet.readthedocs.io/en/latest/?badge=latest)
 
-
 ![Combined outputs](doc/source/_static/PredictingStructures-1.jpg?raw=true "Combined outputs")
 
-## Important Note!
-This repo is currently in development and may not work as expected. For the code corresponding to our Nature Methods paper, please use the `release_1` branch [here](https://github.com/AllenCellModeling/pytorch_fnet/tree/release_1).
+## Support
+
+This code is in active development and is used within our organization. We are currently not supporting this code for external use and are simply releasing the code to the community AS IS. The community is welcome to submit issues, but you should not expect an active response.
 
-## System Requirements
-Installing on Linux is recommended (we have used Ubuntu 16.04).
+For the code corresponding to our Nature Methods paper, please use the `release_1` branch [here](https://github.com/AllenCellModeling/pytorch_fnet/tree/release_1).
 
-An nVIDIA graphics card with >10GB of ram (we have used an nVIDIA Titan X Pascal) with current drivers installed (we have used nVIDIA driver version 390.48).
+## System requirements
+
+We recommend installation on Linux and an NVIDIA graphics card with 10+ GB of RAM (e.g., NVIDIA Titan X Pascal) with the latest drivers installed.
 
 ## Installation
-### Environment setup
-- Note: We no longer provide Conda files, but do recommend installing in a clean python 3.6 environment.
+
+- We recommend an environment manager such as [Conda](https://docs.conda.io/en/latest/miniconda.html).
+- Install Python 3.6+ if necessary.
 - Clone and install the repo:
+
 ```shell
```

Comment:
This is a little pedantic on my part but for 
sh, bash, csh `> pip install .[dev] ` works but 
for zsh I found I had to include some escapes `> pip install \.\[dev\]` 
You might want to specify bash some way maybe `bash prompt> pip install .[dev]`

Reply:
I guess we could specify that all command-line instructions assume the shell is bash.